### PR TITLE
OSDOCS-7426 Z-stream RNs 4.13.9

### PR DESCRIPTION
**Pull request by cbippley**
Version(s):
4.13

Issue:
[OSDOCS-7426](https://issues.redhat.com/browse/OSDOCS-7426)

Link to docs preview:
[Preview](https://63484--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes#ocp-4-13-9)

QE review:
- [ ] QE has approved this change.

Additional information:
Expected to ship live on Aug. 16th. Changed a small typo in the 4.13.8 RNs.

### CODE REVIEW

**Review by skopacz1 — APPROVED** [review]
LGTM! Please wait until @adellape approves this review before proceeding.

**Review by adellape — APPROVED** [review]
@skopacz1 @cbippley 👍

**Pull request merged**
